### PR TITLE
Remind to add 'careportal' to Nightscout

### PR DESCRIPTION
**Pull request by lukasz-kaniowski**
Uploading treatments requires environment variable 'ENABLE' to include 'careportal'.

After scratching my head for some time I've finally figure out why `POST` to `treatments` was not working.

### CODE REVIEW

File: docs/docs/walkthrough/phase-1/visualization.md [hunk 1]
```diff
@@ -118,6 +118,8 @@ Some things to be aware of:
 
 ### Uploading Latest Treatments to Nightscout
 
+Note: Remember to add `careportal` to Nigtscouts `ENABLE` environment variable in case it is not already there.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-1/visualization.md]: "Nigtscouts" should be "Nightscout's"

**Comment by lukasz-kaniowski** [comment]
Thx, this is fixed now.

File: docs/docs/walkthrough/phase-1/visualization.md [hunk 2]
```diff
@@ -118,6 +118,8 @@ Some things to be aware of:
 
 ### Uploading Latest Treatments to Nightscout
 
+Note: Remember to add `careportal` to Nigtscout's `ENABLE` environment variable in case it is not already there.
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-1/visualization.md]: Still missing the "h" in Nightscout's

Comment (lukasz-kaniowski) [inline docs/docs/walkthrough/phase-1/visualization.md]: Embarrassing 😊 Will fix it once I'm home tonight.

Comment (lukasz-kaniowski) [inline docs/docs/walkthrough/phase-1/visualization.md]: ok, now should be fine.

**Pull request merged**